app/src/main/java/com/example/androidenhancedvideoplayer/MainActivity.kt

```diff
@@ -3,18 +3,47 @@ package com.example.androidenhancedvideoplayer
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import com.example.androidenhancedvideoplayer.components.RecommendedVideoComponent
 import com.example.androidenhancedvideoplayer.ui.theme.AndroidEnhancedVideoPlayerTheme
+import com.example.androidenhancedvideoplayer.utils.fillMaxSizeOnLandscape
 import com.profusion.androidenhancedvideoplayer.components.EnhancedVideoPlayer
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             AndroidEnhancedVideoPlayerTheme {
-                EnhancedVideoPlayer(
-                    resourceId = R.raw.login_screen_background,
-                    alwaysRepeat = false
-                )
+                val orientation = LocalConfiguration.current.orientation
+                val playlist = List(5) { it + 1 }
+                Column(
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    Box(
+                        modifier = Modifier
+                            .fillMaxSizeOnLandscape(orientation = orientation)
+                            .fillMaxWidth()
+                    ) {
+                        EnhancedVideoPlayer(
+                            resourceId = R.raw.login_screen_background,
+                            expandContent = true,
+                            alwaysRepeat = false
+                        )
+                    }
+                    LazyColumn {
+                        items(playlist) {
+                            RecommendedVideoComponent(
+                                videoTitle = "Mock Recommended Video $it"
+                            )
+                        }
+                    }
```

Comment:
Move this into the `RecommendedVideoComponent`, and remove the `playlist` from this component as well